lib/miq_automation_engine/engine/miq_ae_event.rb

```diff
@@ -122,14 +123,13 @@ def self.process_result(ae_result, aevent)
         inputs[:host]                  = Host.find_by(:id => aevent[:host_id])               unless aevent[:host_id].nil?
         inputs[:ext_management_system] = ExtManagementSystem.find_by(:id => aevent[:ems_id]) unless aevent[:ems_id].nil?
       end
-
       target     = inputs.delete(:target) || inputs['vm']
       event_name = args['logical_event'] || aevent[:event_type]
-      $miq_ae_logger.info("Enforcing Policy [#{ae_result}]")
+      $miq_ae_logger.info("Enforcing Policy [#{ae_result}]", :resource_id => miq_request_id)
```

Comment:
I don't think this one needs to change, because policies shouldn't have requests associated.